eve/main.yml

```diff
@@ -132,6 +132,9 @@ stages:
         - '/home/eve/workspace'
     steps:
       - Git: *clone
+      - ShellCommand:
+          name: keyscan
+          command: ssh-keyscan -H github.com > ~/.ssh/known_hosts
```

Comment:
related?

Reply:
I realized an easier way to get around this.